ykrt/src/compile/jitc_yk/codegen/x86_64/mod.rs

```diff
@@ -10,19 +10,21 @@
 
 use super::{
     super::{
-        gdb::GdbCtx,
         jit_ir::{self, BinOp, FuncDeclIdx, InstIdx, Operand, Ty},
         CompilationError,
     },
     abs_stack::AbstractStack,
     reg_alloc::{LocalAlloc, RegisterAllocator, StackDirection},
     CodeGen,
 };
+#[cfg(any(debug_assertions, test))]
```

Comment:
Not something for this PR, but I wondered if this should be enabled only when `yk_testing` is enabled?

Reply:
True.